pyaedt/modules/PostProcessor.py

```diff
@@ -4911,3 +4916,454 @@ def sample_ami_waveform(
+
+        Returns
+        -------
+        dict
+            Depending on the quantity chosen, the output dictionary can contain the following keys:
+
+                - ``"Min"``, ``"Max"``, ``"Mean"``, ``"Stdev"``, and ``Unit``
+                - ``"Total"`` and ``Unit``
+
+        References
+        ----------
+
+        >>> oModule.ExportFieldsSummary
+        """
+        fs = self.create_field_summary()
+        fs.add_calculation(
+            "Boundary",
+            ["Surface", "Volume"][int(volume)],
+            boundary_name,
+            quantity_name,
+            side=side,
+            ref_temperature=ref_temperature,
+        )
+        return self._parse_field_summary_content(fs, setup_name, design_variation, quantity_name)
+
+    @pyaedt_function_handler()
+    def evaluate_monitor_quantity(
+        self,
+        monitor_name,
+        quantity_name,
+        side="Default",
+        setup_name=None,
+        design_variation={},
+        ref_temperature="",
+    ):
+        """Export monitor field output.
+
+        Parameters
+        ----------
+        monitor_name : str
+            Name of boundary where the computation is to be performed.
+        quantity_name : str
+            Name of the quantity to export.
+        side : str, optional
+            Select which mesh faces side to use. Available options are ``"Default"``, ``"Adjacent"``,
+            and ``"Combined"``. Default is ``"Default"``.
+        setup_name : str, optional
+            Name of the setup and name of the sweep. For example, ``"IcepakSetup1 : SteatyState"``.
+            The default is ``None``, in which case the active setup and active sweep are used.
+        design_variation : dict, optional
+            Dictionary of parameters defined for the specific setup with values. The default is ``{}``.
+        ref_temperature: str, optional
+            Reference temperature to use for heat transfer coefficient computation. The default is ``""``.
+
+        Returns
+        -------
+        dict
+            Depending on the quantity chosen, the output dictionary can contain the following keys:
+
+                - ``"Min"``, ``"Max"``, ``"Mean"``, ``"Stdev"``, and ``Unit``
+                - ``"Total"`` and ``Unit``
+
+        References
+        ----------
+
+        >>> oModule.ExportFieldsSummary
+        """
+        if settings.aedt_version < "2024.1":
+            raise NotImplementedError("Monitors are not supported in field summary in versions lower than 2024R1.")
+        else:  # pragma: no cover
+            if self._app.monitor.face_monitors.get(monitor_name, None):
+                field_type = "Surface"
+            elif self._app.monitor.point_monitors.get(monitor_name, None):
+                field_type = "Volume"
+            else:
+                raise AttributeError("Monitor {} not found in the design.".format(monitor_name))
```

Comment:
```suggestion
                raise AttributeError("Monitor {} is not found in the design.".format(monitor_name).)
```
Messages should be complete sentence so adding the verb and concluding punctuation.